pkgs/development/tools/build-managers/shards/default.nix

```diff
@@ -36,17 +36,18 @@ let
 
 in
 rec {
+  # needed for anything that requires the old v1 shards format
   shards_0_11 = generic {
     version = "0.11.1";
     sha256 = "05qnhc23xbmicdl4fwyxfpcvd8jq4inzh6v7jsjjw4n76vzb1f71";
     crystal = crystal_0_34;
   };
 
-  shards_0_12 = generic {
+  shards_0_13 = generic {
     version = "0.12.0";
     sha256 = "0dginczw1gc5qlb9k4b6ldxzqz8n97jrrnjvj3mm9wcdbc9j6h3c";
-    crystal = crystal_0_35;
+    crystal = crystal_0_36;
   };
 
-  shards = shards_0_12;
```

Comment:
Why drop 0.12 and leave 0.11?

A quick `rg` shows no usage of either but if the intention of version splitting was to keep the most recent ones I think it would be better to drop 0.11 if any.

```
rg shards_0
pkgs/development/tools/build-managers/shards/default.nix
39:  shards_0_11 = generic {
45:  shards_0_12 = generic {
51:  shards = shards_0_12;

pkgs/top-level/all-packages.nix
12751:    shards_0_11
12752:    shards_0_12
```

Reply:
There is a comment in this PR that addresses exactly that:
`# needed for anything that requires the old v1 shards format`

So it's not for `nixpkgs` per se, but for anybody with a project that hasn't yet been moved to the new v2 shards format.

Reply:
A right, so v0.11 is the last v1 shards format.